app/src/main/res/drawable/category_group_background.xml

```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="utf-8"?>
 <shape xmlns:android="http://schemas.android.com/apk/res/android"
     android:shape="rectangle">
-    <corners android:radius="4dp" />
+    <corners android:radius="8dp" />
```

Comment:
If you see this radius being used as a standard (I see it as a radius in 2 files in this PR), could be useful to extract it to a dimen so you're not updating every spot every time this changes

Reply:
That is a fair point, I pulled those out into the `values/dimens` file. Thanks for the feedback!